tests/integration/suite/webhooks-configs.go

```diff
@@ -0,0 +1,344 @@
+package suite
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"time"
+
+	"github.com/formancehq/formance-sdk-go"
+	. "github.com/formancehq/stack/tests/integration/internal"
+	webhooks "github.com/formancehq/webhooks/pkg"
+	"github.com/formancehq/webhooks/pkg/security"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Given("empty environment for webhooks configs", func() {
+	When("inserting 0 config", func() {
+		Context("getting all configs without filters", func() {
+			It("should return 0 config", func() {
+				resp, httpResp, err := Client().WebhooksApi.
+					GetManyConfigs(TestContext()).Execute()
+				Expect(err).NotTo(HaveOccurred())
+				Expect(resp.Cursor.HasMore).To(BeFalse())
+				Expect(resp.Cursor.Data).To(BeEmpty())
+				Expect(httpResp.StatusCode).To(Equal(http.StatusOK))
+			})
+		})
+	})
+
+	When("inserting 1 config", func() {
+		secret := webhooks.NewSecret()
+		var insertResp *formance.ConfigResponse
+		BeforeEach(func() {
+			var err error
+			var httpResp *http.Response
+			cfg := formance.ConfigUser{
+				Endpoint: "https://example.com",
+				Secret:   &secret,
+				EventTypes: []string{
+					"ledger.committed_transactions",
+				},
+			}
+			insertResp, httpResp, err = Client().WebhooksApi.
+				InsertConfig(TestContext()).ConfigUser(cfg).Execute()
+			Expect(err).To(BeNil())
+			Expect(httpResp.StatusCode).To(Equal(http.StatusOK))
+			Expect(insertResp.Data.Endpoint).To(Equal(cfg.Endpoint))
+			Expect(insertResp.Data.EventTypes).To(Equal(cfg.EventTypes))
+			Expect(insertResp.Data.Active).To(BeTrue())
+			Expect(time.Until(insertResp.Data.CreatedAt)).To(BeNumerically("<", time.Second))
+			Expect(time.Until(insertResp.Data.UpdatedAt)).To(BeNumerically("<", time.Second))
+			_, err = uuid.Parse(insertResp.Data.Id)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		Context("getting all configs without filters", func() {
```

Comment:
Find a way to factorize common setup for common tests.
You can use https://onsi.github.io/ginkgo/#table-specs for example